packages/salesforcedx-vscode-lwc/src/messages/i18n.ts

```diff
@@ -51,6 +51,8 @@ export const messages = {
     'SFDX: Navigate to Lightning Web Component Test',
   no_lwc_jest_found_text:
     'sfdx-lwc-jest is not installed. Install it from https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.unit_testing_using_jest_installation',
+  no_lwc_testrunner_found_text:
```

Comment:
This exposes a url to an internal repo, not sure if that's common practice. @hbc-bindu @ekapner do you have any suggestions here ?

Reply:
I would say we don't want to expose an internal repo. @lcampos -- Want to make sure I understand your question.  And then we can decide next steps. Can they follow the instrux and be successful? If so, we can update the topic to remove references to the internal repo. If they must have access to the repo to be successful, that's a problem. And I would need to discuss with Stephanie and @hbc-bindu what the alternatives are.

Reply:
I've removed the url for now (It's gated anyway). @ekapner @hbc-bindu Let me know what you think with the current text. We will provide them with documentation in internal channels.